### PR TITLE
Add authoritative IAM policy binding for KMS Crypto Keys

### PR DESCRIPTION
**Pull request by maroux**
Fixes terraform-providers/terraform-provider-google#4163

### CODE REVIEW

**Comment by danawillow** [comment]
Hey @maroux, sorry for the delay! You still interested in getting this merged? If so, would you mind resolving merge conflicts?

**Comment by maroux** [comment]
@danawillow updated.

File: website/docs/r/google_kms_crypto_key_iam.html.markdown [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    condition {
+      title       = "expires_after_2019_12_31"
+      description = "Expiring at midnight of 2019-12-31"
+      expression  = "request.time < timestamp(\"2020-01-01T00:00:00Z\")"
+    }
+  }
+}
+```
+
+# google\_kms\_crypto\_key\_iam\_binding
```
[the remark this answers is not in the document]
Comment (maroux) [reply website/docs/r/google_kms_crypto_key_iam.html.markdown]: done.

[the remark this answers is not in the document]
Comment (danawillow) [reply website/docs/r/google_kms_crypto_key_iam.html.markdown]: Almost! The examples themselves changed, not just the addition of the conditions ones.

[the remark this answers is not in the document]
Comment (maroux) [reply website/docs/r/google_kms_crypto_key_iam.html.markdown]: Ah, yes, I added back the real resources. As well as consistent email.

**Comment by danawillow** [comment]
I ended up having to make some small changes to this so I merged the downstream from MM (https://github.com/terraform-providers/terraform-provider-google/pull/5247) instead. You still get commit credit in MM though :)

**Pull request closed without merging**